lib/__tests__/helpers/createMockManager.js

```diff
@@ -7,6 +7,7 @@ module.exports = function() {
     openMenu: jest.fn(),
     handleKeyDown: jest.fn(),
     handleClick: jest.fn(),
+    handleBlur: jest.fn(),
```

Comment:
That's right. I mentioned above that I needed to update createMockManager because tests are failing and I cannot add a new test. I'll separate this into a new PR.